ui/lib/core/addon/helpers/replication-mode-description.js

```diff
@@ -11,7 +11,7 @@ const REPLICATION_MODE_DESCRIPTIONS = {
     'Performance Replication scales workloads horizontally across clusters to make requests faster. Local secondaries handle read requests but forward writes to the primary to be handled.',
 };
 
-export function replicationModeDescription([mode]) {
```

Comment:
this wasn't imported anywhere so i removed it. we're only using the default export on line 18